frontend/website/docs/coda-testnet.md

```diff
@@ -17,6 +17,18 @@ BONUS: An additional 2000 pts to the first person to complete the challenge, and
 
 **Challenge #3: Join Discord** - 100 pts awarded for introducing yourself in the #testnet-general channel. Name, location and what you’re excited about are all good things to share in order to get the points!
 
+**Challenge #4 (on-going): Community MVP** - Each week, we will recognize the winners of the previous week based on the point values below. We may give out all the awards in a week, or none, or several at each level. The more active the community is, the more points* we can award in this category.
```

Comment:
I think you need a backslash before the star (and to make it a link that goes to the anchor on the bottom)

Reply:
ah - just copied this from announcements, ok will fix